web/server/entrypoint.go

```diff
@@ -115,7 +119,20 @@ func RunServer(ctx context.Context, args []string, _ logging.Logger) (err error)
 		return dumpResourceRegistrations(argsParsed.DumpResourcesPath)
 	}
 
-	logger, registry := logging.NewLoggerWithRegistry("rdk")
+	logger, registry := logging.NewBlankLoggerWithRegistry("rdk")
+	// Dan: We changed from a constructor that defaulted to INFO to `NewBlankLoggerWithRegistry`
+	// which defaults to DEBUG. We pessimistically set the level to INFO to ensure parity. Though I
+	// expect `InitLoggingSettings` will always put the logger into the right state without any
+	// observable side-effects.
+	logger.SetLevel(logging.INFO)
+	if argsParsed.OutputLogFile != "" {
+		logWriter, closer := logging.NewFileAppender(argsParsed.OutputLogFile)
+		defer closer.Close()
```

Comment:
This lumberjack library spawns a goroutine that it does not wait on when exiting. Despite this call to close.

I talked with @cheukt and we're cool with just disabling leak detection in goutils for `ContextualMain` (e.g: viam-server and presumably viam-cli). It'll still run for tests.

Reply:
SGTM too.